### PR TITLE
applications: nrf_desktop: Add support for nRF54L PDK (documentation)

### PR DESCRIPTION
**Pull request by MarekPieta**
PR adds documentation for nRF54L PDK support in nRF Desktop.

Jira: NCSDK-26216

### CODE REVIEW

**Comment by MarekPieta** [comment]
Hi @annwoj, @peknis, @divipillai,
I added a draft of doc changes. If preferred, we could merge code now and merge documentation alignments after the NCS release.

**Comment by annwoj** [comment]
> Hi @annwoj, @peknis, @divipillai, I added a draft of doc changes. If preferred, we could merge code now and merge documentation alignments after the NCS release.

I'd suggest merging the doc part after the NCS release, if possible

File: applications/nrf_desktop/board_configuration.rst [hunk 1]
```diff
@@ -101,3 +101,15 @@ Sample dongle (``nrf5340dk_nrf5340``)
       * Bluetooth uses Nordic Semiconductor's SoftDevice link layer without LLPM and is configured to act as a central.
         Input data comes from Bluetooth and is retransmitted to USB.
       * The configuration with the B0 bootloader is set as default.
+
+Sample mouse or keyboard (``nrf54l15pdk_nrf54l15_cpuapp``)
+      * The configuration uses the nRF54L15 Preview Development Kit.
```
Comment (peknis) [inline applications/nrf_desktop/board_configuration.rst:106]:
```suggestion
      * The configuration uses the nRF54L15 Preview Development Kit (PDK).
```

File: applications/nrf_desktop/board_configuration.rst [hunk 2]
```diff
@@ -101,3 +101,15 @@ Sample dongle (``nrf5340dk_nrf5340``)
...
+      * The build types allow to build the application as mouse or keyboard.
```
Comment (peknis) [inline applications/nrf_desktop/board_configuration.rst:107]:
```suggestion
      * The build types allow to build the application as a mouse or a keyboard.
```

File: applications/nrf_desktop/board_configuration.rst [hunk 3]
```diff
@@ -101,3 +101,15 @@ Sample dongle (``nrf5340dk_nrf5340``)
...
+      * Inputs are simulated based on the hardware button presses.
+        On the PDK v0.2.1, GPIOs assigned to **Button 3** and **Button 4** do not support interrupts.
+        Because of this, the application cannot use those buttons.
+      * Only Bluetooth LE transport is enabled.
+        Bluetooth LE is configured to use Nordic Semiconductor's SoftDevice Link Layer and LLPM.
```
Comment (peknis) [inline applications/nrf_desktop/board_configuration.rst:112]:
```suggestion
        Bluetooth LE is configured to use Nordic Semiconductor's SoftDevice Link Layer and Low Latency Packet Mode (LLPM).
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -354,6 +354,9 @@ nRF Desktop
   * The pin control (:kconfig:option:`CONFIG_PINCTRL`) dependency to the :ref:`nrf_desktop_wheel`.
     The dependency simplifies accessing **A** and **B** QDEC pins in the wheel module's implementation.
     The pin control is selected by the QDEC driver (:kconfig:option:`CONFIG_QDEC_NRFX`).
+  * Added support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` board target.
+    The DK can act as a sample mouse or keyboard.
```
Comment (peknis) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:358]: PDK or DK?

**Review by peknis — CHANGES_REQUESTED** [review]

File: applications/nrf_desktop/board_configuration.rst [hunk 5]
```diff
@@ -101,3 +101,15 @@ Sample dongle (``nrf5340dk_nrf5340``)
       * Bluetooth uses Nordic Semiconductor's SoftDevice link layer without LLPM and is configured to act as a central.
         Input data comes from Bluetooth and is retransmitted to USB.
       * The configuration with the B0 bootloader is set as default.
+
+Sample mouse or keyboard (``nrf54l15pdk_nrf54l15_cpuapp``)
+      * The configuration uses the nRF54L15 Preview Development Kit.
+      * The build types allow to build the application as mouse or keyboard.
+      * Inputs are simulated based on the hardware button presses.
+        On the PDK v0.2.1, GPIOs assigned to **Button 3** and **Button 4** do not support interrupts.
```
Comment (annwoj) [inline applications/nrf_desktop/board_configuration.rst:109]:
```suggestion
        On the PDK PCA10056, revision v0.2.1 AB0-ES5, GPIOs assigned to **Button 3** and **Button 4** do not support interrupts.
```

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst (hunk 4 above)
Comment (MarekPieta) [reply doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:358]: Looking at the other comments: `PDK` - updated

**Comment by MarekPieta** [comment]
Applied doc review comments. I will rebase the PR branch once https://github.com/nrfconnect/sdk-nrf/pull/14364 is merged.

**Review by peknis — APPROVED** [review]

**Review by zycz — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Review by mkapala-nordic — APPROVED** [review]

**Pull request merged**